elements/pfe-clipboard/demo/pfe-clipboard.story.js

```diff
@@ -0,0 +1,49 @@
+import { storiesOf } from "@storybook/polymer";
+import * as bridge from "@storybook/addon-knobs/polymer";
+import * as tools from "../../../.storybook/utils.js";
+
+import PfeClipboard from "../dist/pfe-clipboard";
+
+const stories = storiesOf("Clipboard", module);
+
+// Define the template to be used
+const template = (data = {}) => {
+  return tools.component(PfeClipboard.tag, data.prop, data.slots);
+};
+
+// Use these to get dynamically generated content
+// const defaultHeading = tools.autoHeading(true);
+const defaultContent = tools.autoContent(1, 2);
```

Comment:
You might need to peek at the storybook for this component and confirm that this dummy content works for you.  I suspect it will be too verbose and you'll likely just want a quick, "Copy to clipboard".

Reply:
Updated the storybook to have better defaults!

https://user-images.githubusercontent.com/3428964/103700078-e95cbd80-4f71-11eb-9610-52c19a823b65.mov